README.md

```diff
@@ -44,19 +45,21 @@ This table only lists the options used in this role, see [xen-vman](https://gith
+| `hypervisor_host`      | `{{ vm_facts_default_hypervisor_host }}`                                                                                                                                                                                                            | Hypervisor host to use for this VM.                                                                                                                                                                                                                                                                                                                        |
+| `backup_host`          | `{{ vm_facts_default_backup_host }}`                                                                                                                                                                                                                | Host to which this VM shall be backed up to.                                                                                                                                                                                                                                                                                                               |
+| `pull_storage_from`    | :heavy_multiplication_x:                                                                                                                                                                                                                            | This will add a list item to the generated `vm_facts_move_storages` var, consisting of a dict containing the keys `source_storage`, `source_dataset`, `target_storage`, `target_dataset_suffix`, `source_backup_dataset`, `target_backup_dataset` and `backup_host`  needed to move a VM.                                                                  |
+| `pull_hypervisor_from` | :heavy_multiplication_x:                                                                                                                                                                                                                            | This will add a list item to the generated `vm_facts_move_hypervisors` var, consisting of a dict containing the keys `source_hypervisor`, `destination_hypervisor`, `org` and `host` needed to move a VM.                                                                                                                                                  |
```

Comment:
Please add an example or describe valid values for both options.

Reply:
You should enter the hostname of the respective source host. I clarified the description in another commit.